notebook/tests/selenium/test_deletecell.py

```diff
@@ -56,3 +55,6 @@ def test_delete_cells(notebook):
     notebook.current_cell.send_keys('cv')
     assert len(notebook.cells) == 2
     assert cell_is_deletable(notebook, 1)
+
+    remove_cells(notebook)
```

Comment:
Careful! If you look at the code just above this, it's making one of the cells undeletable. So we're not actually deleting all the cells here.

Reply:
Thanks for reviewing, changed the code according to the comments.